src/library.js

```diff
@@ -706,16 +706,21 @@ LibraryManager.library = {
 
   // TODO: Initialize these to defaults on startup from system settings.
   // Note: glibc has one fewer underscore for all of these. Also used in other related functions (timegm)
-  tzset__proxy: 'sync',
+  mktime__deps: ['tzset_impl'],
```

Comment:
`mktime__deps` already exists on line 540, maybe update that one instead (good to keep the deps alongside the implementation)... and actually I think deps of mktime are unchanged by this PR aren't they?

Reply:
@sbc100   Sorry, the deps should have been for tzet, not mktime.   I have since corrected.
I'll watch for the checks to succeed.